op5build/ninja.logrotate

```diff
@@ -0,0 +1,22 @@
+/var/log/op5/ninja.log {
+    su monitor apache
```

Comment:
This file is apache:apache not monitor:apache.

Reply:
This is still wrong.

Reply:
ohh sorry. Updated correct file now. Thnx